src/main/resources/org/rust/ide/liveTemplates/other.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version='1.0' encoding='UTF-8'?>
+
+<templateSet group="Rust">
+    <template name="f" description="field" toReformat="true" toShortenFQNames="true"
+              value="$IDENT$: $TYPE$,">
+        <variable name="IDENT" expression="" defaultValue="&quot;foo&quot;" alwaysStopAt="true"/>
+        <variable name="TYPE" expression="" defaultValue="&quot;u32&quot;" alwaysStopAt="true"/>
+        <context>
+            <option name="RUST_STRUCT" value="true"/>
+        </context>
+    </template>
+
+    <template name="pf" description="pub field" toReformat="true" toShortenFQNames="true"
+              value="pub $IDENT$: $TYPE$,">
+        <variable name="IDENT" expression="" defaultValue="&quot;foo&quot;" alwaysStopAt="true"/>
+        <variable name="TYPE" expression="" defaultValue="&quot;u32&quot;" alwaysStopAt="true"/>
+        <context>
+            <option name="RUST_STRUCT" value="true"/>
+        </context>
+    </template>
+
+    <template name="main" description="main() function declaration" toReformat="true" toShortenFQNames="true"
+              value="fn main() {&#10;    $END$&#10;}">
+        <context>
+            <option name="RUST_FILE" value="true"/>
+            <option name="RUST_STATEMENT" value="false"/>
+            <option name="RUST_ITEM" value="false"/>
+            <option name="RUST_MOD" value="true"/>
```

Comment:
I've tried it, and `rustc` requires to place a `#[main]` attribute on it and then yells that it is unstable. Let's just remove it if  nobody uses it?